core/indigo-core/molecule/molecule_json_loader.h

```diff
@@ -42,17 +43,17 @@ namespace indigo
     class QueryMolecule;
     class SGroup;
     class MetaObjectsInterface;
+    using namespace rapidjson;
```

Comment:
Please avoid `using namespace ..` in header files on the global level.

Reply:
Fixed